cmd/rest-server/listener_windows.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net"
+	"os"
+)
+
+// findListener creates a listener.
+func findListener(inetd bool, addr string) (listener net.Listener, err error) {
+	if inetd {
+		log.Print("inetd mode activated")
```

Comment:
Can this even work on windows?

Reply:
Via cygwin maybe? Or WSL? It compiled, so I left it there...